run.py

```
@@ -0,0 +1,6 @@
+from app import create_app
+
+app = create_app()
+
+if __name__ == "__main__":
+    app.run(debug=True, host="0.0.0.0", port=3000)
```

Ruff (0.15.15)

[error] Possible binding to all interfaces

(S104)